projects/plugins/jetpack/_inc/client/newsletter/subscriptions-settings.jsx

```diff
@@ -95,12 +97,16 @@ function SubscriptionsSettings( props ) {
 			return '';
 		}
 
+		const source = isWpAdminInterface
```

Comment:
I think all Jetpack sites can get the Cloud version, does this already do that?

For the Atomic sites, any reason not to link to cloud from Jetpack settings, too?

That would make the check unnecessary, allow you to update `calypso-subscribers` target and all old Jetpack versions would get the cloud version too.

Reply:
> I think all Jetpack sites can get the Cloud version, does this already do that?
> For the Atomic sites, any reason not to link to cloud from Jetpack settings, too?

That's a great point! I chatted with @allilevine and @lupus2k about it and we decided to stick with limiting to Classic enabled sites for now to "limit the blast radius." Then we can follow up opening it up to all Atomic.